tensorflow_io/core/kernels/audio_kernels.cc

```diff
@@ -19,8 +19,86 @@ limitations under the License.
 
 namespace tensorflow {
 namespace data {
+
+// DecodedAudio
+size_t DecodedAudio::data_size() {
+  return channels * samples_perchannel * sizeof(int16);
```

Comment:
I think google's code style prefer method name with CamelCase, so DataSize instead?

Also, not all audio files stay with `int16` so this one has to at least take into consideration the data types. But that is a larger discussion.